packages/mattermost-redux/src/action_types/apps.ts

```diff
@@ -7,7 +7,10 @@ import keyMirror from 'mattermost-redux/utils/key_mirror';
 // susceptible to breaking changes without pushing the major version of this package.
 export default keyMirror({
     RECEIVED_APP_BINDINGS: null,
+    FAILED_TO_FETCH_APP_BINDINGS: null,
     RECEIVED_APP_RHS_BINDINGS: null,
     RECEIVED_APP_COMMAND_FORM: null,
     RECEIVED_APP_RHS_COMMAND_FORM: null,
```

Comment:
@larkox Do we need to add `FAILED_TO_FETCH_APP_RHS_BINDINGS` etc?

Reply:
If I remember correctly, we are cleaning up before fetching the RHS data, so should not be needed. Nevertheless, I wonder if we better be safe than sorry, specially if we want to interpret a failure as "the plugin is not working anymore".

Also, we could keep the same action for both. I don't see material difference between failing to get the bindings on the RHS and failing to get the bindings on the main channel.